packages/app-store/ee/routing-forms/playwright/tests/basic.e2e.ts

```diff
@@ -106,6 +107,59 @@ test.describe("Routing Forms", () => {
       await page.isVisible("text=ERROR 404");
     });
 
+    // TODO: Move these tests to @calcom/app-store/typeform
+    test.describe("Typeform App", () => {
+      test("should copy link in editing area", async ({ page, context }) => {
+        await page.waitForSelector('[data-testid="empty-screen"]');
+        context.grantPermissions(["clipboard-read", "clipboard-write"]);
+        await page.goto(`/apps/typeform`);
+        await page.click('[data-testid="install-app-button"]');
+        await page.waitForNavigation({
+          url: (url) => url.pathname === `/apps/typeform/how-to-use`,
+        });
+
+        await page.goto(`/apps/routing-forms/forms`);
+        const formId = await addForm(page);
+        await fillForm(page, {
+          description: "",
+          field: { label: "test", typeIndex: 1 },
+        });
+
+        await page.click('button[tooltip="Delete"] ~ button[data-state="closed"]');
```

Comment:
Shouldn't rely on the order of Delete being present before the button. This could target wrong element easily in future.
Let's try to add data-testid directly to the button being clicked